R/exceptions.R

```diff
@@ -33,4 +33,6 @@
     warnings
 }
 
-
+`str.Rcpp_stack_trace` <- function(x, ...) {
```

Comment:
Not a fan of the ` style for functions, personally.  Add little here but oh well -- we have a mixed bag of styles in the code already.